examples/input/gamepad_rumble.rs

```diff
@@ -28,16 +28,16 @@ fn gamepad_system(
         };
 
         if button_pressed(GamepadButtonType::South) {
```

Comment:
Nit: The previous comment recommended the first `if` to be the `North` condition, the second to be `East`, `South`, `West`. Well, maybe it's a bit ethnocentric, but I always read N E S W in this sequence, and it was surprising to me and more difficult to interpret the code because it didn't follow this expectation.